tests/end2end/playwright/form_advanced.spec.js

```diff
@@ -0,0 +1,114 @@
+import { test, expect } from '@playwright/test';
+import { ProjectPage } from './pages/project';
+
+let project;
+
+test.describe('Advanced form', function () {
+
+    test.beforeEach(async function ({ page }) {
+        project = new ProjectPage(page, 'form_advanced');
+        await project.open();
+
+        project.openEditingFormWithLayer('form_advanced_point');
+
+        // Click on map as form needs a geometry
+        project.clickOnMapLegacy(410, 175);
+    });
+
+    test('should toggle tab visibility when toggling checkbox', async function ({
+        page,
+    }) {
+        await expect(
+            page.locator('#jforms_view_edition-tabs > li:nth-child(2)')
+        ).not.toBeVisible();
+        await expect(
+            page.locator('#jforms_view_edition-tabs > li:nth-child(2)')
+        ).toHaveText('photo');
```

Comment:
The `photo` tab is visible by default, it seems we last somthing about group visibilities at form display.

The `getGroupvisibilities` request always responses `{"jforms_view_edition-tab1":true,"jforms_view_edition-tab2":true}` because the request only contains : `__form`, `__formid` and `__JFORMS_TOKEN__`.